### DISCUSSION TITLE
Mtex for python

### DISCUSSION THREAD

**Discussion by YamaguS** [mtex-toolbox/mtex#1609] [Ask Anything]
I think Mtex for python is mayflower-mtex.
But, alomost no documents were not found.
Does anyone have it ?

**Comment by ralfHielscher** [comment]
Could you please be a little bit more specific with you request?  I was not able to find mayflower-mtex.
In general it is possible to call matlab from python.

Ralf.

**Reply by YamaguS** [reply]
Thanks for your reply, and sorry for mistake.
I ment matflow-mtex.
https://pypi.org/project/matflow-mtex/

**Reply by YamaguS** [reply]
If it is possible, please show me how to call matlab from python.

**Comment by lisakemp** [comment]
Generically, this is how you call MATLAB from Python:
https://www.mathworks.com/help/matlab/matlab_external/call-matlab-functions-from-python.html
